swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md

```diff
@@ -99,27 +119,39 @@ password = ""
 database = ""
 ```
 
-Additionally, you can set values for the following advanced configuration parameters in the `Config.toml` file in your project to use the MySQL data store. Please refer to the [MySQL Connector documentation](https://lib.ballerina.io/ballerinax/mysql/1.8.0#Options) for more information on these parameters.
+Additionally, you can set values for the following advanced configuration parameters in the `Config.toml` file in your project to use the MySQL data store. Please refer to the [MySQL Connector documentation](https://lib.ballerina.io/ballerinax/mysql/latest#Options) for more information on these parameters.
+
+#### MSSQL
+
+```toml
+[<packageName>.<moduleName>]
+host = "localhost"
+port = 1433
+user = "sa"
+password = ""
+database = ""
+```
+
+Additionally, you can set values for the following advanced configuration parameters in the `Config.toml` file in your project to use the MSSQL data store. Please refer to the [MSSQL Connector documentation](https://lib.ballerina.io/ballerinax/mssql/latest#Options) for more information on these parameters.
```

Comment:
```suggestion
Additionally, you can set values for the following advanced configuration parameters in the `Config.toml` file in your project to use the MSSQL data store. Please refer to the [MSSQL Connector documentation](https://lib.ballerina.io/ballerinax/mssql/latest#Options) for more information on these parameters.
```
Refer the writing guidelines on how to add hyperlinks.

Reply:
Fixed in [d8d5667](https://github.com/ballerina-platform/ballerina-dev-website/pull/7147/commits/d8d5667f5300667aac35aaa9bfb7a0194be20ad4)